src/client/containers/RegistrationPage/StudentRegistrationPage/StudentRegistrationPage.component.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+// import { AppHeader } from '../../../components/Appheader/AppHeader.component';
+// import Vector from '../../../components/Appheader/Vector.svg';
+import { Avatar } from '../../../components/Avatar/Avatar';
+import student from '../../../assets/images/student.png';
+import Input from '../../../components/Input/Input';
+import DropDown from '../../../components/Dropdown/DropDown';
+import { Button } from '../../../components/Button/Button';
+import logo from '../../../assets/images/hyf-logo.png';
+import './StudentRegistrationPage.styles.css';
+
+// TODO: Array values would be updated through the DB
+const classes = [
+  {
+    id: '01',
+    value: 'Class14',
+  },
+  {
+    id: '02',
+    value: 'Class15',
+  },
+  {
+    id: '03',
+    value: 'Class16',
+  },
+];
+
+export const StudentRegistrationPage = () => {
+  const [name, setName] = useState('');
+  const [hyfclass, sethyfClass] = useState('Class...');
```

Comment:
The default value should be the first value in the above `classes` so...set it to `Class14`. If you look at the UI in action, it is preselected.

Reply:
done